cafebabel/static/css/styles.css

```diff
@@ -1643,7 +1643,7 @@ textarea#body {
 }
 
 .article-language-selector li.original-language a:before {
-  content: 'Original language';
+  content: attr(title);
```

Comment:
Whaaaatt??? Oh My holly F*** G**! I had no clue we could do that in CSS!
💣

Reply:
haha ! 🚀